### PR TITLE
Enabled score display, new game start, and my profile in NavBar

### PR DESCRIPTION
**Pull request by haruka-izm**
The styling will be implemented later.

### CODE REVIEW

**Review by Flaagrah — APPROVED** [review]
Look good.

**Review by pavel6767 — COMMENTED** [review]
looks good, I would remove unused imports from navbar and gameoutcome

File: client/src/components/NavBar.jsx [hunk 1]
```diff
@@ -1,58 +1,148 @@
...
+import React, { Fragment } from "react";
 import { useHistory } from "react-router-dom";
 import { Typography, Button, Box } from "@material-ui/core";
+import Avatar from "@material-ui/core/Avatar";
+import PersonIcon from "@material-ui/icons/Person";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import ListItem from "@material-ui/core/ListItem";
+import Grid from "@material-ui/core/Grid";
+import { withStyles } from "@material-ui/core/styles";
```
Comment (pavel6767) [inline client/src/components/NavBar.jsx]: unused import

File: client/src/components/NavBar.jsx [hunk 2]
```diff
@@ -1,58 +1,148 @@
...
+    fontFamily: "Roboto",
+  },
+  bar: {
+    fontSize: "20px",
+    fontWeight: "bold",
+  },
 });
 
-export default function (props) {
+const styles = (theme) => {};
```
Comment (pavel6767) [inline client/src/components/NavBar.jsx]: unused variable

File: client/src/components/protected/GameOutcome.jsx [hunk 3]
```diff
@@ -5,6 +5,8 @@ import Button from "@material-ui/core/Button";
 import DialogTitle from "@material-ui/core/DialogTitle";
 import Dialog from "@material-ui/core/Dialog";
 import { withRouter } from "react-router-dom";
+//import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
```
Comment (pavel6767) [inline client/src/components/protected/GameOutcome.jsx]: remove comments

**Review by pavel6767 — APPROVED** [review]

**Pull request merged**